content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -14,20 +14,21 @@ Defines a [data source](/concepts/components/prisma-schema/data-sources) <span c
-| `url`                  | **Yes**  | String (URL)                                                     | Connection URL including authentication info. Most connectors use [the syntax provided by the database](/reference/database-reference/connection-urls#format). |
-| `shadowDatabaseUrl`    | No       | String (URL)                                                     | Connection URL to the shadow database used by Prisma Migrate. Allows you to use a cloud-hosted database as the shadow database.                                |
-| `referentialIntegrity` | No       | String (`foreignKeys`, `prisma`)                                 | Allows setting the [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity).                                                |
+| Name                   | Required | Type                                                                            | Description                                                                                                                                                    |
+| ---------------------- | -------- | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `provider`             | **Yes**  | String (`postgresql`, `mysql`, `sqlite`, `sqlserver`, `mongodb`, `cockroachdb`) | Describes which data source connectors to use.                                                                                                                 |
+| `url`                  | **Yes**  | String (URL)                                                                    | Connection URL including authentication info. Most connectors use [the syntax provided by the database](/reference/database-reference/connection-urls#format). |
+| `shadowDatabaseUrl`    | No       | String (URL)                                                                    | Connection URL to the shadow database used by Prisma Migrate. Allows you to use a cloud-hosted database as the shadow database.                                |
+| `referentialIntegrity` | No       | String (`foreignKeys`, `prisma`)                                                | Allows setting the [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity).                                                |
 
 The following providers are available:
 
-- `sqlite`
-- `postgresql`
-- `mysql`
+- [`sqlite`](/concepts/database-connectors/sqlite)
+- [`postgresql`](/concepts/database-connectors/postgresql)
+- [`mysql`](/concepts/database-connectors/mysql)
 - [`sqlserver`](/concepts/database-connectors/sql-server)
-- `mongodb`
+- [`mongodb`](/concepts/database-connectors/mongodb) (Preview)
```

Comment:
GA now, might need to merge the most main version in this PR

Reply:
Thanks, should be fixed now